serverless.yml

```diff
@@ -257,6 +257,8 @@ functions:
           caching:
             enabled: true
             ttlInSeconds: 3600
+            cacheKeyParameters:
+              - name: request.header.Origin
```

Comment:
Are you sure the problem will only occur in this endpoint?

There are other endpoints called by the Explorer, they will still be caching the CORS headers and may present the same problem, right?

Reply:
@luislhl The problem will occur on all endpoints since we have cors enabled on all of them.

We should check that we can add the `cacheKeyParameters` on the global `apiGatewayCaching` and that it will not conflict with the other apis that have their own `cacheKeyParameters`. If there is a conflict, we can just add the header on each api.